packages/application-generic/src/services/workers/active-jobs-metric-worker.service.ts

```diff
@@ -9,6 +9,5 @@ const LOG_CONTEXT = 'ActiveJobsMetricWorkerService';
 export class ActiveJobsMetricWorkerService extends WorkerBaseService {
   constructor() {
     super(JobTopicNameEnum.ACTIVE_JOBS_METRIC);
-    Logger.log(`Worker ${this.topic} instantiated`, LOG_CONTEXT);
```

Comment:
removed this duplicated log, we already log that the worker was initiated in the initWorker() in the base class, we could add something else here if needed.